.github/workflows/deploy_sdk.yml

```diff
@@ -0,0 +1,60 @@
+# This is a workflow for automatically generating
+# multiple SDK, based on swagger docs
+name: Generate SKD
+
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      IS_CI: true
+      CI_COMMIT_AUTHOR: Continuous Integration
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.19
+
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Install CLI for documentation generation
+        run: go install github.com/swaggo/swag/cmd/swag@latest
+
+      - name: Generate OpenAPI documentation
+        run: swag init
+
+      - name: Build typescript client
+        run: |
+          docker run -u $(id -u):$(id -g) --network=host --rm --workdir=/local \
+          -v $(pwd):/local openapitools/openapi-generator-cli generate \
+          -i docs/swagger.json \
+          -g typescript \
+          -o dist/typescript-client \
+          --package-name=jojo-discord-bot-api-client
+
+      - name: Build go client
+        run: |
+          docker run -u $(id -u):$(id -g) --network=host --rm --workdir=/local \
+          -v $(pwd):/local openapitools/openapi-generator-cli generate \
+          -i docs/swagger.json \
+          -g go \
+          -o dist/go-client \
+          --package-name=jojo-discord-bot-api-client
+
+      - name: Push code into external sdk repository
+        run: |
+          git config --global user.name "${{ env.CI_COMMIT_AUTHOR }}"
+          git config --global user.email "lazybytez-cloud@users.noreply.github.com"
```

Comment:
Please add the id of the noreply address. Newer GitHub accounts always also habe an ID infront.

Reply:
Yeah, what is the ID?

Reply:
```suggestion
          git config --global user.email "83846103+lazybytez-cloud@users.noreply.github.com"
```